lib/std/packedsets.nim

```diff
@@ -557,52 +557,62 @@ proc disjoint*[A](s1, s2: PackedSet[A]): bool =
 
 proc card*[A](s: PackedSet[A]): int {.inline.} =
   ## Alias for `len() <#len,PackedSet[A]>`_.
+  ##
+  ## Card stands for the [cardinality](http://en.wikipedia.org/wiki/Cardinality)
+  ## of a set.
   result = s.len()
 
 proc `<=`*[A](s1, s2: PackedSet[A]): bool =
-  ## Returns true if `s1` is subset of `s2`.
+  ## Returns true if `s1` is a subset of `s2`.
   ##
-  ## A subset `s1` has all of its elements in `s2`, and `s2` doesn't necessarily
+  ## A subset `s1` has all of its elements in `s2`, but `s2` doesn't necessarily
   ## have more elements than `s1`. That is, `s1` can be equal to `s2`.
   runnableExamples:
-    var
-      a = initPackedSet[int]()
-      b = initPackedSet[int]()
-    a.incl(1)
-    b.incl(1); b.incl(2)
-    assert a <= b
-    a.incl(2)
+    let
+      a = toPackedSet([1])
+      b = toPackedSet([1, 2])
+      c = toPackedSet([1, 2, 3])
     assert a <= b
-    a.incl(3)
-    assert(not (a <= b))
+    assert b <= b
+    assert(not (c <= b))
 
   for item in s1.items:
     if not s2.contains(item):
       return false
   return true
 
 proc `<`*[A](s1, s2: PackedSet[A]): bool =
-  ## Returns true if `s1` is proper subset of `s2`.
+  ## Returns true if `s1` is a proper subset of `s2`.
   ##
   ## A strict or proper subset `s1` has all of its elements in `s2`, but `s2` has
   ## more elements than `s1`.
   runnableExamples:
-    var
-      a = initPackedSet[int]()
-      b = initPackedSet[int]()
-    a.incl(1)
-    b.incl(1); b.incl(2)
+    let
+      a = toPackedSet([1])
+      b = toPackedSet([1, 2])
+      c = toPackedSet([1, 2, 3])
     assert a < b
-    a.incl(2)
-    assert(not (a < b))
+    assert(not (b < b))
+    assert(not (c < b))
+
   return s1 <= s2 and not (s2 <= s1)
 
 proc `==`*[A](s1, s2: PackedSet[A]): bool =
   ## Returns true if both `s1` and `s2` have the same elements and set size.
+  runnableExamples:
+    let
+      a = toPackedSet([1, 2])
+      b = toPackedSet([2, 1])
+    assert a == b
+
```

Comment:
avoid variables used just once, makes for shorter, more readable tests
and here maybe add 2nd line (ya i know, it's now used twice but simplicity argument still applies)
```suggestion
    assert [1, 2].toPackedSet == [2, 1].toPackedSet
    assert [1, 2].toPackedSet == [2, 1, 2].toPackedSet

```

Reply:
I kept the variables in some places, since I think they make it more readable there.